File: csharp/Production/utils/DirectionHandler.cs
using System;

namespace utils
{
    public static class DirectionHandler{

        public static String getString(this Direction dir)
        {              // Getter
            switch (dir)
            {
                case Direction.CW:
                    return "CW";
                case Direction.CCW:
                    return "CCW";
                default:
                    return "ERROR";
            }

        }

        static public Direction getDirection(int intValue)
        {
            if (intValue == 0)
                return Direction.CW;
            else
                return Direction.CCW;
        }

        static public Direction getDirection(String intValue)
        {
            if (intValue == "0")
                return Direction.CW;
            else
                return Direction.CCW;
        }
     

    static public Direction getOpposite(Direction direction) {
        if (direction == Direction.CW)
            return Direction.CCW;
        else
            return Direction.CW;
    }
}
}

File: python/Production/utils/DirectionHandler.py
from production.utils.direction import Direction
class DirectionHandler:

    @staticmethod
    def getString(thisdir):              # Getter
        if thisdir == Direction.CW:
            return "CW"
        else:
                return "CCW"
            

    

    @staticmethod
    def getDirectionInt(intValue):
        if intValue == 0:
            return Direction.CW
        else:
            return Direction.CCW
    

    @staticmethod
    def getDirectionString(intValue):
        if intValue == "0":
            return Direction.CW
        else:
            return Direction.CCW
    
    

    @staticmethod
    def getOpposite(direction):
        if direction == Direction.CW:
            return Direction.CCW
        else:
            return Direction.CW



